src/components/Video/__tests__/Video.test.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Video, reducedMotionMediaQuery } from '../Video';
+
+function mockMatchMedia(
+  mediaQuery: string = '',
+  matches: boolean = false
+): void {
+  Object.defineProperty(window, 'matchMedia', {
+    writable: true,
+    value: jest.fn().mockImplementation((query: string = mediaQuery) => ({
+      matches: matches,
+      media: query,
+      onchange: null,
+      addEventListener: jest.fn(),
+      removeEventListener: jest.fn(),
+      dispatchEvent: jest.fn()
+    }))
+  });
+}
+
+// Mock HTMLMediaElement.prototype.pause
+jest
+  .spyOn(window.HTMLMediaElement.prototype, 'pause')
+  .mockImplementation(() => {});
+
+const playSpy = jest
```

Comment:
I don't think playSpy is ever being reset, making these tests (callCounts) order dependent.

Reply:
Ah thanks, I think you're right, and also explains why I was having trouble with a test earlier. In the latest I added a:
```
afterEach(() => {
    jest.clearAllMocks();
  }); 
```
And updated the tests to be a little more specific (one tests with the media query set to true, one now explicitly sets it to false in the test).